admin_guide/managing_projects.adoc

```diff
@@ -10,12 +10,19 @@
 toc::[]
 
 == Overview
+
+In OpenShift, projects are used to isolate content from groups of developers. As an OpenShift administrator, you can give developers access to certain projects, allow them to create their own, and give them administrator rights.
+
+[[selfprovisioning-projects]]
+
+== Self-provisioning Projects
+
 You can allow developers to create their own projects. There is an endpoint
 that will provision a project according to a
 link:../dev_guide/templates.html[template]. The web console and `oc new-project` 
 command use this endpoint when a developer link:../dev_guide/projects.html[creates a new project].
 
-== Template for New Projects
+=== Template for New Projects
```

Comment:
Minor nit, but possibly update this to be a gerund phrase, and add a manual anchor:

```
[[modifying-the-template-for-new-projects]]
=== Modifying the Template for New Projects
```